src/main/java/com/microsoft/lst_bench/telemetry/JDBCTelemetryRegistry.java

```diff
@@ -94,6 +94,7 @@ public void flush() {
                   o ->
                       String.join(
                           ",",
+                          o.getExperimentRunId().toString(),
```

Comment:
Bug: this sql statement will fail to execute as the experiment run it string is not quoted

Reply:
Fixed.